### PR TITLE
Update readme for auction sample

### PR DESCRIPTION
**Pull request by nikhil550**
Signed-off-by: Nikhil Gupta <ngupta@symbridge.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,6 @@ transfer an asset in a more realistic transfer scenario.
 | [State-Based Endorsement](asset-transfer-sbe) | This sample demonstrates how to override the chaincode-level endorsement policy to set endorsement policies at the key-level (data/asset level). | [Using State-based endorsement](https://github.com/hyperledger/fabric-samples/tree/master/asset-transfer-sbe) | Java, TypeScript | JavaScript |
 | [Secured agreement](asset-transfer-secured-agreement) | Smart contract that uses implicit private data collections, state-based endorsement, and organization-based ownership and access control to keep data private and securely transfer an asset with the consent of both the current owner and buyer. | [Secured asset transfer](https://hyperledger-fabric.readthedocs.io/en/latest/secured_asset_transfer/secured_private_asset_transfer_tutorial.html)  | Go | JavaScript |
 
-Additional smart contract and application languages are still being developed. The series will also be integrated with other Fabric tutorials in the near future. For more information, see the public plan for [Fabric samples improvements](https://docs.google.com/presentation/d/1UxK2HH8SrQyZU58MnuDb9hr1nmekst8b/edit#slide=id.g776cdbfb06_0_51).
```
Comment (nikhil550) [inline README.md:34]: We can remove this because the series is largely complete, an all the tutorials have been rebased

File: README.md [hunk 2]
```diff
@@ -31,7 +31,6 @@ transfer an asset in a more realistic transfer scenario.
 | [State-Based Endorsement](asset-transfer-sbe) | This sample demonstrates how to override the chaincode-level endorsement policy to set endorsement policies at the key-level (data/asset level). | [Using State-based endorsement](https://github.com/hyperledger/fabric-samples/tree/master/asset-transfer-sbe) | Java, TypeScript | JavaScript |
```
Comment (nikhil550) [inline README.md:31]: Should we add the transfer-events sample to this series? It wasn't quite sure if it fit.

Comment (denyeart) [reply README.md:31]: Yes, I've just merged a PR that adds the asset-transfer-events readme, we can add it to the root readme in a subsequent PR.

File: README.md (hunk 1 above)
Comment (denyeart) [reply README.md:34]: Agreed.

**Review by denyeart — APPROVED** [review]

**Pull request merged**